sympy/concrete/summations.py

```diff
@@ -450,8 +450,18 @@ def is_convergent(self):
             if lim_evaluated > 1:
                 return S.false
 
-        ### ------------- alternating series test ----------- ###
+        ### ----------- Raabe's test ---------------- ###
+        lim_val = Limit(sym * (sequence_term/sequence_term.subs(sym, sym + 1)
```

Comment:
There was `simplify` on this line at some point. Did that make some of the currently failing tests pass?